client/rest-high-level/src/test/java/org/elasticsearch/client/rollup/DeleteRollupJobResponseTests.java

```diff
@@ -18,34 +18,25 @@
  */
 package org.elasticsearch.client.rollup;
 
-import org.elasticsearch.common.xcontent.XContentParser;
-import org.elasticsearch.test.AbstractXContentTestCase;
-import org.junit.Before;
+import org.elasticsearch.client.core.AcknowledgedResponseTests;
+import org.elasticsearch.test.ESTestCase;
 
 import java.io.IOException;
 
-public class DeleteRollupJobResponseTests extends AbstractXContentTestCase<DeleteRollupJobResponse> {
+import static org.elasticsearch.test.AbstractXContentTestCase.xContentTester;
 
-    private boolean acknowledged;
+public class DeleteRollupJobResponseTests extends ESTestCase {
```

Comment:
@hub-cap What do you think about removing `DeleteRollupJobResponse` and `PutRollupJobResponse` classes? These classes don't add anything and use the same `acknowledged` field as in the `AcknowledgedResponse` base class.

Reply:
I think we would have to deprecate those rather than just delete them. But im in favor of consolidating them down to the `AcknowledgedResponse` class

Reply:
Cool. I've removed the two response classes.